core/ir/aarch64/codec.txt

```diff
@@ -1158,20 +1160,23 @@ x101101011000000000101xxxxxxxxxx  cls     wx0 : wx5
 1001111000111000000000xxxxxxxxxx     fcvtzs    x0 : s5
 0001111001111000000000xxxxxxxxxx     fcvtzs    w0 : d5
 1001111001111000000000xxxxxxxxxx     fcvtzs    x0 : d5
-0001111000111001000000xxxxxxxxxx     fcvtzu    w0 : s5
-1001111000111001000000xxxxxxxxxx     fcvtzu    x0 : s5
-0001111001111001000000xxxxxxxxxx     fcvtzu    w0 : d5
-1001111001111001000000xxxxxxxxxx     fcvtzu    x0 : d5
+x001111000111001000000xxxxxxxxxx     fcvtzu    wx0 : s5
```

Comment:
This patch folds the 4 `fcvtzs x0 : <FPreg>5` and `fcvtzs w0 :  <FPreg>5` encodings into 2 `fcvtzs wx0 : <FPreg>5` encodings. A subsequent PR will check and fold others.